autogpt/memory/message_history.py

```diff
@@ -88,8 +88,8 @@ def per_cycle(self, config: Config, messages: list[Message] | None = None):
         """
         Yields:
             Message: a message containing user input
-            Message: a message from the AI containing a proposed action
-            Message: the message containing the result of the AI's proposed action
+            Message: a message from the AI containing a proposed ability
```

Comment:
This doesn't work gramatically. The AI proposes an action to take, or an ability *to use*.